docs/modules/environment.md

```diff
@@ -140,5 +360,132 @@ Determine if it is currently thundering or not.
 
 ---
 
+### Time Predicates
+
+:::::{tab-set}
+::::{tab-item} Day Time
+
+**`bs.environment:is_day_time`**
+
+Determine if it is currently daytime (between 07:00 and 18:00).
+
+> **Credits**: theogiraudet
+
+::::
+::::{tab-item} Night Time
+
+**`bs.environment:is_night_time`**
+
+Determine if it is currently nighttime (between 19:00 and 06:00).
+
+> **Credits**: theogiraudet
+
+::::
+::::{tab-item} Sunrise
+
+**`bs.environment:is_sunrise_time`**
+
+Determine if it is currently sunrise (between 06:00 and 07:00).
+
+> **Credits**: theogiraudet
+
+::::
+::::{tab-item} Sunset
+
+**`bs.environment:is_sunset_time`**
+
+Determine if it is currently sunset (between 18:00 and 19:00).
+
+> **Credits**: theogiraudet
+
+::::
+:::::
+
+---
+
+### Villager Behavior
+
+:::::{tab-set}
+::::{tab-item} Working
+
+**`bs.environment:are_villagers_working`**
```

Comment:
This name imply a state which is not really what is checked. Villagers could be awake in a sleeping phase if there is no bed for example. I suggest:
is_villager_sleep_time
is_villager_work_time
is_villager_meet_time
Or
*_sleeping_time
_working_time
_socializing_time